lib/fluent/plugin/out_amqp.rb

```diff
@@ -53,6 +53,7 @@ class AMQPOutput < Output
 
     config_section :buffer do
       config_set_default :@type, DEFAULT_BUFFER_TYPE
+      config_set_default :chunk_keys, ['tag']
```

Comment:
And this lines ensures that `chunk.metadata` contains `tag`.
Otherwise, `chunk.metadata` does not have any information.